charts/sps-sample/values.yaml

```diff
@@ -0,0 +1,55 @@
+refarch-gateway:
```

Comment:
Die Werte in dieser Datei sind LHM-spezifisch und müssen raus bzw. dies könnte als Beispiel drin bleiben. Es muss aber dokumentiert werden, was die Werte bedeuten und wie diese zu setzen sind.

Reply:
Sehe ich nicht so. Ich will ja, eine funktionierende Anwendung haben. siehe hier https://git.muenchen.de/ccse/cicd/sps/-/issues/609#note_853077

Reply:
Wir arbeiten Open Source. D.h. das Repo muss so aufgebaut sein, dass es auch von nicht-LHM-Projekten genutzt werden kann. Wenn die Werte LHM-spezifisch sind und nicht dokumentiert werden, ist das m.E. kein guter Open Source Code. Wie gesagt: es kann beispielhaft so drin stehen, aber es MUSS so gut dokumentiert werden, dass nicht-LHM Projekte verstehen, für was die Werte stehen, wo sie genutzt werden und wie sie zu konfigurieren sind.

Reply:
nicht lhm spezifisch:
```yml
  env:
    - name: SPRING_PROFILES_ACTIVE
      value: no-security
```      
musste ich wegen https://github.com/it-at-m/refarch/issues/193 und https://github.com/it-at-m/refarch/issues/180 reinmachen
```      
    - name: SSO_ISSUER_URL
      value: https://sso.muenchen.de/auth/realms/muenchen.de
```
nicht lhm spezifisch:
```
    - name: SPRING_CLOUD_GATEWAY_ROUTES_0_ID
      value: backend
    - name: SPRING_CLOUD_GATEWAY_ROUTES_0_URI
      value: "http://a-backend:8080/"
    - name: SPRING_CLOUD_GATEWAY_ROUTES_0_PREDICATES_0
      value: Path=/api/beispielprojekt-backend-service/**
    - name: SPRING_CLOUD_GATEWAY_ROUTES_0_FILTERS_0
      value: RewritePath=/api/beispielprojekt-backend-service/(?<urlsegments>.*), /$\{urlsegments}
    - name: SPRING_CLOUD_GATEWAY_ROUTES_0_FILTERS_1
      value: RemoveResponseHeader=WWW-Authenticate

    - name: SPRING_CLOUD_GATEWAY_ROUTES_1_ID
      value: frontend
    - name: SPRING_CLOUD_GATEWAY_ROUTES_1_URI
      value: "http://a-frontend:8080/"
    - name: SPRING_CLOUD_GATEWAY_ROUTES_1_PREDICATES_0
      value: Path=/**
    - name: SPRING_CLOUD_GATEWAY_ROUTES_1_FILTERS_0
      value: RewritePath=/(?<urlsegments>.*), /$\{urlsegments}
    - name: SPRING_CLOUD_GATEWAY_ROUTES_1_FILTERS_1
      value: RemoveResponseHeader=WWW-Authenticate
```

das ist openshift spezifisch. Keine Ahnung wie das in Kubernetes funktioniert. Es gibt leider kein TestCluster.
 theoretisch kann ich auch einfach beispiel.example.de beim Host eintragen. https://github.com/SonarSource/helm-chart-sonarqube/blob/master/charts/sonarqube/values.yaml#L31 haben die bei Sonarqube auch so gemacht.
````

  ingress:
    enabled: true
    annotations:
      route.openshift.io/termination: "edge"
    className: openshift-default
    hosts:
      - host: sps-frontend-dev-spstest-test.apps.capk.muenchen.de
        paths:
          - path: /
            pathType: "ImplementationSpecific"
            backend:
              serviceName: '{{ include "refarch-gateway.fullname" . }}'
              servicePort: 8080
```

Reply:
@hupling  können wir das so trennen?

Reply:
habe ich doch gemacht. alles lhm spezifische auskommentiert. mit # davor